core/src/main/java/io/questdb/cairo/wal/WalDataRecord.java

```diff
@@ -129,6 +126,29 @@ public long getLong(int col) {
         return reader.getColumn(absoluteColumnIndex).getLong(offset);
     }
 
+    @Override
+    public long getLong128Hi(int col, long location) {
```

Comment:
there is terminology mix here, same thing is called `location` and `offset`. Everywhere else `Unsafe.getUnsafe().getLong()` takes `address`. We should be consistent

Reply:
I intentionally avoided the word `address` - to avoid the temptation (on callers side) to access it directly via Unsafe. in some cases the `location` is not an address. from the caller's perspective, it's just an opaque token. see the contract: https://github.com/questdb/questdb/blob/ec9cfc1a719a847a99d6e9b97c5776494066dfee/core/src/main/java/io/questdb/cairo/sql/Record.java#L192-L200